.github/workflows/release.yaml

```diff
@@ -0,0 +1,32 @@
+name: Release Partner Charts
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Configure git
+      run: |
+        git config --global user.name "github-actions[bot]"
+        git config --global user.email "github-actions[bot]@users.noreply.github.com"
+
+    - name: Update main-source branch
+      run: |
+        scripts/pull-ci-scripts
+        bin/partner-charts-ci auto
```

Comment:
We are skipping `bin/partner-charts-ci validate`  [step](https://github.com/rancher/partner-charts-ci#5-run-a-validation)  to ensure released assets are not being modified, is this step neede now?

Reply:
The purpose of this automation is to release updated assets, i.e. modify them. So we don't want to ensure that they are not modified. Or maybe you know something I don't?

Reply:
Thank you Adam. AFAIK, It ensures that we don't modify the already released assets (not the one that we are updating). In any case, is just to be sure that we are skipping this step consciously.